starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,68 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+  <section id="section-heading">
+  <div class="main-header">
+    <h1>Apple Pies</h1>
```

Comment:
the h1 is not centered in the middle:) Try to fix this by deleting the width in your `.main-header h1` and adding `display: flex` and `justify-content: center` to your `.main-header`